src/Collection/CollectionInterface.php

```diff
@@ -619,6 +619,25 @@ public function first();
      */
     public function last();
 
+    /**
+     * Returns the last N elements of a collection
+     *
+     * ### Examples:
+     *
+     * ```
+     * $items = [1, 2, 3, 4, 5];
+     *
+     * $last = (new Collection($items))->last(3);
```

Comment:
I think this example should say `->lastN(3)`, no?

Reply:
Yes